README.md

```diff
@@ -117,3 +117,37 @@ Read the [Contributing guidelines](/CONTRIBUTING.md) for more information.
 ***
 
 [MIT](/LICENSE) © 2017
+
```

Comment:
As mentioned in comments, this should be moved to documentation at [simplajs/docs](https://github.com/simplajs/docs)

Reply:
Sure will do that.